jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -0,0 +1,61 @@
+job {
+  name 'terraform-deploy'
+  description('Deploy and automatically update Terraform environments upon repository changes')
+  parameters {
+    choiceParam("action", ["apply", "destroy"], "Select if you want to apply changes to environments or destroy them.")
```

Comment:
Could you wrap this line please (and all other lines that go over about 100 characters) as it makes it quite hard to review.